server/commands.go

```diff
@@ -239,7 +275,8 @@ func NewServerCommands(defaultCfg *sconfig.Config) []*cli.Command {
 						},
 					}
 
-					opt, err := newUIOption(uiBaseCfg)
+					loggerAdapter := NewUILoggerAdapter(logger)
```

Comment:
could be worth initializing a second logger for UI instead of adapting server's logger?